deploy/chart/templates/0000_50_olm_03-olmConfig.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operators.coreos.com/v1
+kind: OLMConfig
+metadata:
+  name: cluster
```

Comment:
not sure this needs to be included by default; OLM should probably create a default if it can't find one on the cluster.

nit: we need to make sure this isn't included downstream by default b/c CVO will stomp user changes.

Reply:
We'd need to add the create-only annotation downstream.